cmd/tailscale/callbacks.go

```diff
@@ -42,6 +39,9 @@ var (
 
 	// onWriteStorageGranted is notified when we are granted WRITE_STORAGE_PERMISSION.
 	onWriteStorageGranted = make(chan struct{}, 1)
+
+	// onDNSConfigChanged is notified when the network changes and the DNS config needs to be updated.
+	onDNSConfigChanged = make(chan struct{}, 1)
```

Comment:
The new callback no longer plumbs through the bool for online/offline notifications. Do we still need that, or am I missing something?

Reply:
I actually don't think we ever needed that? We never checked what the bool was, we just did stuff when there was an update.